dapps/fast-usdc/vm-integration/test/fast-usdc.test.ts

```diff
@@ -653,7 +670,9 @@ test.serial('replace operators', async t => {
     runUtils: { EV },
     walletFactoryDriver: wfd,
   } = t.context;
-  const { creatorFacet } = await EV.vat('bootstrap').consumeItem('fastUsdcKit');
+  const { creatorFacet } = (await EV.vat('bootstrap').consumeItem(
```

Comment:
confirm that without the cast it's `unknown` instead of `any`. that will prompt giving it a type